tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "gts/tslint.json","linterOptions": {"exclude": ["src/helpers.ts", "src/index.ts"]}
```

Comment:
I think we do want to include those though...

Reply:
Yes, no need to exclude helpers (now that it's a good well-written TS). We do want to lint it.

Reply:
it's only for`helper.ts` not having a class but uses `this`, tslint keeps complain about that even if with `// tslint:disable-next-line no-any`

Reply:
How exactly does it complain - can you copy&paste the exact `tslint` output?

Reply:
```
// tslint:disable-next-line no-any
return (this as any).annotateImage(annotateImageRequest, callOptions, callback);
```
Throws:
`'this' implicitly has type 'any' because it does not have a type annotation.`